pkg/kyverno/test/test_command.go

```diff
@@ -182,7 +182,7 @@ type TestResults struct {
 	Rule              string              `json:"rule"`
 	Result            report.PolicyResult `json:"result"`
 	Status            report.PolicyResult `json:"status"`
-	Resource          string              `json:"resource"`
+	Resource          []string            `json:"resource"`
```

Comment:
@chipzoller Actually I had to change this to an array to make these changes, so this might not be backwards compatible, is there any way to make this support either a string or an array?

Reply:
@realshuting if this is a breaking change, do we want to proceed?

Reply:
we have to deal with backward compatibility. Many users using old kyverno_test.yaml

Reply:
I have an idea, let me see if it works...it might be we can make it fully backwards compatible

Reply:
> we can support 2 parameters , resource(old) and resources(new) and we can deprecate in next release.

So I looked into how to support backwards compatibility a bit and I think that this might be the way to proceed indeed.

Reply:
@4molybdenum2 is this resolved?